index.js

```diff
@@ -6,7 +6,7 @@ function rewrite(apiPath, extensions, req, next) {
 	const url = req.url.split("?");
 	const b = `${url[0]}/${req.method}`;
 	const mapped = extensions.some(ext => {
-		const resourcePath = `${b}.${ext}`;
+		const resourcePath = url[1] ? `${b}_${url[1]}.${ext}` : `${b}.${ext}`;
```

Comment:
IMHO, this is single-purpose extension that inserts into path the whole query string, which includes potentially unsafe characters: equal sign, percent, ampersand.
https://www.mtu.edu/umc/services/digital/writing/characters-avoid/